server.js

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+import bodyParser from "body-parser";
+import fs from "fs";
+
+const app = express();
+
+app.use(express.static("./public"))
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
```

Comment:
Here we configure express to serve any files in the `public/` directory whose names match the requested URL. We also configure the body parser to parse incoming requests and build a JS object of any values included in the request body.